server/api/expense-pool/expense-pool.controller.js

```diff
@@ -1,12 +1,20 @@
+import { ExpensePool } from '../../database/database';
 const controller = {};
 
 controller.addPool = (req, res) => {
-  // create statement
-  // grab data from request body
-  // sendStatus response
-  res.sendStatus('addPool');
+  ExpensePool.create({
+    name: 'test1',
+    description: 'trying to input a value',
+    closed: false,
+  }, ['name', 'description', 'closed'])
+  .then((expensePool) => {
```

Comment:
remove the `()` around `(expensePool)` to  match similar single arg arrow functions.